access_group.go

```diff
@@ -168,6 +168,12 @@ type AccessGroupLoginMethod struct {
 	} `json:"login_method"`
 }
 
+type AccessGroupDevicePosture struct {
```

Comment:
looks like we're missing a comment here and causing lint to fail https://github.com/cloudflare/cloudflare-go/pull/622/checks?check_run_id=2462609162#step:5:17

Reply:
Updated 👍